go/api/base_client.go

```diff
@@ -1163,6 +1164,33 @@ func (client *baseClient) PTTL(key string) (Result[int64], error) {
 	return handleLongResponse(result)
 }
 
+func (client *baseClient) PfAdd(key string, elements []string) (Result[int64], error) {
+	result, err := client.executeCommand(C.PfAdd, append([]string{key}, elements...))
+	if err != nil {
+		return CreateNilInt64Result(), err
+	}
+
+	return handleLongResponse(result)
+}
+
+func (client *baseClient) PfCount(keys []string) (Result[int64], error) {
+	result, err := client.executeCommand(C.PfCount, keys)
+	if err != nil {
+		return CreateNilInt64Result(), err
+	}
+
+	return handleLongResponse(result)
+}
+
+func (client *baseClient) Select(index int64) (Result[string], error) {
```

Comment:
This must be implemented in standalone client only. Cluster has no databases to switch into, so this command is no-op there.

Reply:
done